tidy.go

```diff
@@ -11,7 +11,7 @@ import (
 // Errors & warnings are deliberately suppressed as underlying tools
 // throw warnings very easily.
 func Tidy(r io.Reader, xmlIn bool) ([]byte, error) {
-	f, err := ioutil.TempFile("/tmp", "sajari-convert-")
+	f, err := NewLocalFile(r)
```

Comment:
I think this is now doing a double copy of the reader. As this will do a copy into the LocalFile, and then bellow this on https://github.com/sajari/docconv/pull/64/files#diff-fdb0944ddcd7e93cbe32976d0e1cfd19L19 we are doing a copy again.

Reply:
fixed